### PR TITLE
Add some details on property config. Fixes #1328.

### LINKED ISSUE #1318: Property docs should mention deserializing JSON values
Currently documented here:

https://www.polymer-project.org/1.0/docs/migration.html#attr

But not, as one might expect, here:

https://www.polymer-project.org/1.0/docs/devguide/properties.html#attribute-deserialization

### LINKED ISSUE #1328: Boolean attributes don't behave as described
If a Boolean attribute is defined as:
``` javascript
properties: {
    active: Boolean
}
```
And a tag is implemented as:
``` html
<example-tag></example-tag>
```
The result is that `active` is `undefined`, not `false` as [described in the documentation](https://www.polymer-project.org/1.0/docs/devguide/properties.html#attribute-serialization).
<hr>
Likewise, if a Boolean value is defined as:
``` javascript
properties: {
    active: {
        type: Boolean,
        value: true
    }
}
```
And used in a tag as:
``` html
<example-tag></example-tag>
```
The value is `true` (not `false` because the attribute isn't assigned), which would run contrary to the same documentation.

It's not clear to me if this is a bug or misleading documentation.

### PR DESCRIPTION
**Pull request by arthurevans**
Also fixes #1318.

### CODE REVIEW

File: 1.0/docs/devguide/properties.md [hunk 1]
```diff
@@ -233,6 +236,19 @@ encouraged that attributes only be used for configuring properties in static
 markup, and instead that properties are set directly for changes at runtime. 
 {: .alert .alert-info }
 
+### Configuring boolean properties
+
+For a Boolean property to be configurable from markup, it must default to `false`. If it defaults to `true`, you cannot set it to `false` from markup, since the presence of the attribute, with or without a value, equates to `true`. This is the standard behavior for attributes in the web platform.
```
Comment (ebidel) [inline 1.0/docs/devguide/properties.md]: standard behavior for attributes -> standard behavior for Boolean attributes

**Comment by ebidel** [comment]
LGTM

**Comment by addyosmani** [comment]
LGTM2

**Comment by arthurevans** [comment]
OK. Merging.

**Pull request merged**